test/pjrt/test_experimental_pjrt_tpu.py

```diff
@@ -33,18 +33,10 @@ def setUp(self):
     # sub-processes will not be able to initialize the client witht the correct
     # settings.
 
+  @absltest.skipIf(tpu.get_tpu_env()['ACCELERATOR_TYPE'] == 'v3-8',
```

Comment:
from resourcing standpoint, is it not safer to target `v2-8` devices as a starting point?

Reply:
If the tests pass on `v2-8`, I support this. Most of our usage is on `v3`, but `v2` is very similar and generally much more available.

I recently added [`tpu.version`](https://github.com/pytorch/xla/blob/2fd2ffeeb303245ccab76cd5287d9818565aaad2/torch_xla/experimental/tpu.py#L91-L98), which will make it easier to skip tests that only work on v4.

Reply:
Let's confirm and transition.

Reply:
Migrated to v2. Leaving in this annotation because we still want to skip this test in the case that it ever does get run on TPU v3.

Reply:
This test will still fail on v2, right? Can you change the skip condition to `tpu.version() <= 3`?

Reply:
Corrected this skip and added a bunch for tests that failed on v2